x-pack/solutions/observability/plugins/infra/public/pages/metrics/inventory_view/components/waffle/node_square.tsx

```diff
@@ -201,7 +203,9 @@ export const NodeSquare = ({
             ellipsisMode && (
               <ValueInner aria-label={nodeAriaLabel}>
                 {}
-                <Label color={color}>...</Label>
+                <Label color={color}>
+                  {i18n.translate('xpack.infra.nodeSquare.label.Label', { defaultMessage: '...' })}
```

Comment:
This does not need to be translated.

Reply:
Ah, that was automatically changed by the linter when saving, I'll fix it